spec/features/create_etd_spec.rb

```diff
@@ -1,36 +1,56 @@
 # frozen_string_literal: true
-# Generated via
-#  `rails generate hyrax:work Etd`
+
 require 'rails_helper'
 include Warden::Test::Helpers
 
-RSpec.feature 'Create an ETD', js: false do
-  let(:title) { 'Comet in Moominland' }
+RSpec.feature 'Create and edit an OSHU ETD', js: false do
   let(:admin) { FactoryBot.create(:admin) }
   let(:user) { FactoryBot.create(:user) }
   let(:etd) do
+    FactoryBot.attributes_for(
+      :moomins_thesis,
+      user: admin,
+      visibility: Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC
+    )
+  end
+  let(:editing_etd) do
```

Comment:
Maybe leave this as `:etd` and name the other variable `:etd_attributes`?

Reply:
It was easier to change this in a few places and seemed just as descriptive, but we are going to want to move editing into its own feature spec as well. Might be best to do that now.